docs/dev/install.rst

```diff
@@ -149,6 +149,13 @@ the endpoint ``/api/v3/``. No user interface is provided.
 
 .. _fork: http://help.github.com/fork-a-repo/
 
+Docker
+______
+
+For local Docker setup please reference the docker_ section.
```

Comment:
Can you change this to make it clearer that this is an alternative setup method?